tests/ResourceSyncingTest.php

```diff
@@ -126,6 +126,34 @@
     ], ResourceUser::first()->getAttributes());
 });
 
+test('creating the resource in tenant database creates it in central database and used custom attributes', function () {
```

Comment:
The test name seems a bit messy

Reply:
`creating the resource in tenant database creates it in central database with provided attribute`

Is this better?